micro_sam/training/util.py

```diff
@@ -111,6 +113,8 @@ def _get_prompt_lists(self, gt, n_samples, prompt_generator):
     def __call__(self, x, y, n_pos, n_neg, get_boxes=False, n_samples=None):
         """Convert the outputs of dataloader and prompt settings to the batch format expected by SAM.
         """
+        # function to apply resizing for the input prompts
+        transform = ResizeLongestSide(1024)
```

Comment:
Let's not hard-code this. I would suggest passing the transform as an argument to `__init__`. (And then you can use the `model.transform` when instantiating the class for training)

Reply:
Okie. Fixed it as well. Thanks!